resources/istio/files/istio-operator-cluster-evaluation.yaml

```diff
@@ -178,7 +171,10 @@ spec:
     accessLogFile: ""
     trustDomain: cluster.local
     defaultConfig:
-      proxyMetadata: {}
+      proxyMetadata:
```

Comment:
in default manifest this is not present, please check if this setting needs to be set specifically

Reply:
This is the setting for the coredns addin replacement. It is necessary.